src/modules/script/script.router.ts

```diff
@@ -1,17 +1,16 @@
 import express, { Request, Response } from 'express';
 import logger from '../../logger';
+import { validation } from '../../validation/validation';
+import { IdNumberDto } from './DTO';
 
 export const scriptRouter = express.Router();
 
 scriptRouter.get('/:numberScript', (req: Request, res: Response) => {
-  const numberScript = req.params.numberScript;
-  if (Number(numberScript) < 100) {
-    res.json({ error: 'Ваш скрипт' });
-    logger.info('Запрос  скрипта');
-  } else {
-    res.json({ error: 'Такого скрипта не существует' });
-    logger.error('Запрос несуществующего скрипта');
-  }
+  const numberScript = { id: Number(req.params.numberScript) };
+
+  validation(IdNumberDto, numberScript);
```

Comment:
И путь сменить на `/:id`
```suggestion
  const dto =  validation(IdNumberDto, req.params);
```